src/sysinfo

```diff
@@ -910,6 +922,21 @@ if [[ ${ZONENAME} == "global" ]]; then
         echo '  "Zpool Size in GiB":' ${Zpool_size}','
     fi
 
+    if [[ -n "has_recovery" ]]; then
```

Comment:
`ouput_parseable()` should have something like this too.

Reply:
That'd probably better to be fixed with OS-7859 since the rest of the zpool stuff is missing from the parseable output (I actually have a change w/ most of that done already, but is missing the recovery bits since it predates them).  Either way the `sysinfo -p` isn't needed by anything for EDAR (the agents use the JSON output).